.appveyor.yml

```diff
@@ -1,11 +1,15 @@
 version: "{build}"
 clone_folder: c:\gopath\src\github.com\mackerelio\mkr
 environment:
+  VERSION: go1.14.1
   GOPATH: c:\gopath
-  GOROOT: c:\go112
+  GOROOT: c:\go
   GO111MODULE: on
 build: off
 install:
+  - rmdir %GOROOT% /s /q
+  - appveyor DownloadFile https://dl.google.com/go/%VERSION%.windows-amd64.zip
```

Comment:
Can/should we cache this file? (Or does it fast enough?)

Reply:
Though AppVeyor has cache directive, I'm not sure how to skip downloading the archive on windows batch script! sorry.